package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "jstransformer-coffee-script",
-  "version": "1.1.1",
+  "name": "jstransformer-coffee",
```

Comment:
Can we stick to the old name. That way when people upgrade their dependencies, they will automatically get upgraded, rather than having to manually seek out the new package.